crates/lovely-core/src/patch/module.rs

```diff
@@ -17,6 +17,8 @@ pub struct ModulePatch {
     // If enabled, evaluate the module immediately upon loading it
     #[serde(default)]
     pub load_now: bool,
+    #[serde(skip)]
+    pub display_source: String,
```

Comment:
Add a comment here on what this field does so future contributors don't need to dig through the source